libherokubuildpack/src/buildpack_output/mod.rs

```diff
@@ -227,10 +227,6 @@ where
         prefix_first_rest_lines(Self::PREFIX_FIRST, Self::PREFIX_REST, s.as_ref())
     }
 
-    pub fn mut_step(&mut self, s: impl AsRef<str>) {
-        writeln_now(&mut self.state.write, Self::style(s));
-    }
```

Comment:
This served a purpose: when we want to log in a context that we cannot easily get the log from, this allows us to emit an output without changing the state.